app-config.yaml

```diff
@@ -51,8 +51,10 @@ proxy:
   #     target: 'https://example.com'
   #     changeOrigin: true
   endpoints:
-    '/kafka-lag':
+    '/kafka-lag-stage':
```

Comment:
I don't love this naming scheme. It feels like it should be 

`
'kafka-lag/stage`:
     ...
`kafka-lag/prod`:
    ....
```